release/models/network-instance/openconfig-network-instance-l2.yang

```diff
@@ -376,4 +383,308 @@ submodule openconfig-network-instance-l2 {
+         that is learned and installed into the MAC VRF Layer 2 Routing
+         Information Base (L2RIB)";
+
+      container mac-table {
+        description
+          "Oper state container for MAC address information installed into
+           the MAC VRF of the L2RIB";
+
+        container entries {
+          description
+            "Enclosing container for list of MAC address entries";
+
+          list entry {
+            key "mac-address vlan";
+            description "List of learned MAC addresses";
+
+            leaf mac-address {
+              type leafref {
+                path "../state/mac-address";
+              }
+              description "Leafref of MAC address object";
+            }
+
+            leaf vlan {
+              type leafref {
+                path "../state/vlan";
+              }
+              description
+                "Leafref VLAN on which the MAC address is present";
+            }
+
+            container state {
+              config false;
+              description
+                "Operational state data for L2RIB MAC table object entry";
+              uses l2ni-l2rib-mac-table-state;
+            }
+
+            container producers {
+              description "Source producers for each MAC Table entry";
+              list producer {
+                key "producer";
+                description
+                  "List of producers for each MAC table entry";
+
+                leaf producer {
+                  type leafref {
+                    path "../state/producer";
+                  }
+                  description
+                    "Reference to producer list key";
+                }
+
+                container state {
+                  config false;
+                  description "State container for L2RIB MAC Table Entries";
+                  uses l2ni-l2rib-common-producer-state;
+                  uses l2ni-l2rib-mac-table-producer-state;
+                }
+              }
+            }
+
+            uses l2ni-l2rib-common-next-hop-state;
+          }
+        }
+
+      }
+
+      container mac-ip-table {
+        description
+          "Oper state container for MAC-IP address information installed into
+           the MAC VRF of the L2RIB";
+
+        container entries {
+          description
+            "Enclosing container for list of MAC-IP address entries";
+
+          list entry {
+            key "mac-address vlan host-ip";
+            description "List of learned MAC-IP addresses";
+
+            leaf mac-address {
+              type leafref {
+                path "../state/mac-address";
+              }
+              description "Leafref of MAC-IP address object";
+            }
+
+            leaf vlan {
+              type leafref {
+                path "../state/vlan";
+              }
+              description
+                "Leafref VLAN on which the MAC-IP address is present";
+            }
+
+            leaf host-ip {
```

Comment:
What is the meaning of the host-ip address. 
 Can you please expand the description.

Reply:
I will update the description based on info in https://datatracker.ietf.org/doc/html/rfc7432

Reply:
Reference added